### PR TITLE
docs on agents examples and readme

### PR DESCRIPTION
**Pull request by stefania11**
Initial pass on agent deploy documentation + small updates on agent examples

### CODE REVIEW

**Comment by mdwelsh** [comment]
I think we want all documentation to be on the docs.fixie.ai site and the README here should point to that site, rather than have to maintain docs in two different places. Would you mind incorporating this documentation into the existing fixie-sdk docs instead?

**Comment by stefania11** [comment]
User study participants all requested for full instructions on the repo. So
I will keep it here and also add it on docs.

**Comment by mdwelsh** [comment]
I don't want to maintain docs in two places, so please, let's not do this. We already have too many pieces of inconsistent documentation and lacking someone whose full time job it is to maintain it, I'd prefer to keep it all in one place. Appreciate your help updating it but please do so in the SDK.

File: README.md [hunk 1]
```diff
@@ -12,12 +12,67 @@ Install the Fixie CLI with `pip install fixieai`. Once installed, run `fixie aut
...
 Deploying agents will automatically upload your agent to the Fixie cloud and start serving it immediately on the platform.
 
-Running `fixie agent deploy` will create (or update) the agent and upload your fewshots and functions to Fixie.
+Running `fixie agent deploy` will create (or update) the agent and upload your fewshots and functions to Fixie. 
+Once you ran this command you can go to app.fixie.ai and test your new agent by calling it in the main chat with "@agentname". 
+
+
+
+## Styleguide for agents
+We recommend that you keep the agent handle and agent name consistent and capitalize it. See example below
```
Comment (juberti) [inline README.md:62]: out of date

Comment (stefania11) [reply README.md]: fixed

**Pull request merged**